app/main/helpers/csv_generator.py

```diff
@@ -0,0 +1,22 @@
+import unicodecsv
+
+
+def iter_csv(rows, headers):
```

Comment:
Good work on moving this into here. 

You'll notice we have a very similar function in the [buyer frontend](https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/5b1f1640166b13f47fc8feacb52bc4a09c677302/app/buyers/views/buyers.py#L317). The question now is should this be moved into dm-utils? I'd probably say no for the moment just so we don't creep the scope of this story but maybe something for the future.